packages/components/src/components/card/card.scss

```diff
@@ -1,14 +1,11 @@
 .dc-card {
```

Comment:
Did you also check all the components using this to make sure these changes does not break anything in them?

Reply:
Already checked. The component has been added by me recently so it needed a couple of changes in action.